src/css/footer.css

```diff
@@ -10,6 +10,103 @@ footer.footer {
   margin: 0.5rem 0;
 }
 
-.footer a {
-  color: var(--footer-link-font-color);
+.footer-info {
+  display: flex;
+  gap: 24px;
+}
+
+@media screen and (max-width: 720px) {
+  .footer-info {
+    flex-direction: column;
+  }
+}
+
+@media screen and (min-width: 720px) {
+  .footer-info {
+    flex-direction: row;
+  }
+}
+
+.footer-info-item-large {
+  flex-grow: 3;
+}
+
+.footer-info-item {
+  flex-grow: 1;
+}
+
+.footer-info-item h2 {
+  margin-top: 0;
+}
+
+.footer-link-list {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.footer-link-list li a {
+  color: var(--footer-font-color);
+}
+
+.footer-link-list li a:hover {
+  text-decoration: none;
+  color: var(--color-brand-primary);
+}
+
+.c-and-socials {
+  display: flex;
+  gap: 24px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+@media screen and (max-width: 720px) {
+  .c-and-socials {
+    flex-direction: column;
+  }
+}
+
+@media screen and (min-width: 720px) {
+  .c-and-socials {
+    flex-direction: row;
+  }
+}
+
+.footer-container {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.social-icons-container {
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+}
+
+.social-icon {
+  background-color: var(--footer-font-color);
+  color: var(--body-font-color);
+  border-radius: 50%;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 20px;
+  aspect-ratio: 1;
```

Comment:
**Optional:** `aspect-ratio` is widely supported but unfortunately not yet in the browser I am using ([qutebrowser](https://qutebrowser.org/)). You can call it fate or you could support the users of non-mainstream browsers by using `width` instead.

Reply:
It must be fate :smile: Wider support is better, I'll fix it.

Reply:
[220e26a](https://github.com/stackabletech/documentation-ui/pull/31/commits/220e26a2173a5d43c4c4f56abc04ccaffebab069)